README.md

```diff
@@ -141,6 +141,21 @@ by a string formatted as:
 
 `<protocol>://[<username>:<password>@]<fqdn>:<port>`
 
+### Timer jobs
+UA client sets up a systemd timer to run jobs that need to be executed recurrently.
+Everytime the timer runs, it decides which jobs need to be executed based on their
+_intervals_. Current jobs being checked and executed are:
+
+| Job | Description | Interval |
+| --- | ----------- | -------- |
+| update_messaging | Update MOTD and APT messages | 6 hours |
+| update_status | Update UA status | 12 hours |
+| gcp_auto_attach | Try to auto-attach a GCP instance | 30 minutes |
```

Comment:
I think you mean `Try to auto-attach on a GCP instance`.

Also, I think we should better describe why we are running each job, specially the GCP one.